lib/matplotlib/text.py

```diff
@@ -279,6 +279,44 @@ def _get_multialignment(self):
         else:
             return self._horizontalalignment
 
+    def char_index_at(self, x):
```

Comment:
```suggestion
    def _char_index_at(self, x):
```
Let's keep this private for now. This implementation does not handle multi-line strings (For that case it only considers characters in the first row). That's good enough for an input line, but not good enough to be public API.